lib/view/artwork_common_widget.dart

```diff
@@ -1630,3 +1630,62 @@ class FeralfileArtworkDetailsMetadataSection extends StatelessWidget {
     );
   }
 }
+
+class ExpandedWidget extends StatefulWidget {
+  final Widget? header;
+  final Widget? child;
+
+  const ExpandedWidget({Key? key, this.header, this.child}) : super(key: key);
+
+  @override
+  State<ExpandedWidget> createState() => _ExpandedWidgetState();
+}
+
+class _ExpandedWidgetState extends State<ExpandedWidget> {
+  bool _isExpanded = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Column(
```

Comment:
I think we don't need an additional Column widget outside the `GestureDetector`, looks like it's redundant.